spec.bs

```diff
@@ -2823,6 +2824,12 @@ might choose to require a [=k-anonymity=] threshold of fifty users over a seven
 will maintain the count over the chosen duration and compare the count to the chosen [=k-anonymity=]
 threshold when responding to [=query k-anonymity count=].
 
+The [=user agent=] should maintain a <dfn>k-anonymity cache</dfn> as a [=map=]
+whose [=map/keys=] are k-anonymity keys for all of the ads and ad components in

```

Comment:
What is a "k-anonymity" key? I'd expect to see a type here.

Reply:
`[=SHA-256=]`?